src/Admin/BaseFieldDescription.php

```diff
@@ -149,8 +158,25 @@ public function __construct(?string $name = null, array $options = [])
         }
 
         $this->setOptions($options);
+
+        if ([] !== $fieldMapping) {
+            $this->setFieldMapping($fieldMapping);
+        }
+
+        if ([] !== $associationMapping) {
+            $this->setAssociationMapping($associationMapping);
+        }
+
+        if ([] !== $parentAssociationMappings) {
+            $this->setParentAssociationMappings($parentAssociationMappings);
+        }
     }
 
+    // NEXT_MAJOR: Uncomment the following lines.
+    // abstract public function setFieldMapping(array $fieldMapping): void;
+    // abstract public function setAssociationMapping(array $associationMapping): void;
+    // abstract public function setParentAssociationMappings(array $parentAssociationMappings): void;
```

Comment:
```suggestion
    // abstract protected function setFieldMapping(array $fieldMapping): void;
    // abstract protected function setAssociationMapping(array $associationMapping): void;
    // abstract protected function setParentAssociationMappings(array $parentAssociationMappings): void;
```

If I understand correctly ?

Reply:
oops, yes. Fixed